package-res/resources/web/common-ui-require-js-cfg.js

```diff
@@ -63,6 +64,14 @@
   requirePaths["pentaho/_globalContextVars"] = basePath + "/pentaho/_globalContextVars";
   requirePaths["pentaho/contextVars"] = requirePaths["pentaho/_globalContextVars"];
 
+  // Singleton interface, virtual module ids
+  //requireMap["*"]["pentaho/spec/IContext"] = "pentaho/service!pentaho.spec.IContext?single";
+  requireMap["*"]["pentaho/config/IService"] = "pentaho/service!pentaho.config.IService?single";
```

Comment:
I also see now why it is confusing...
1. `pentaho/config/IService.instance`
2. `pentaho/config/IService.the`
3. `pentaho/config/IService.first`
4. `pentaho/config/IService.default`
5. `pentaho/config/iService` 
6. `pentaho/config/the.IService`

I'll create a poll for this on slack.